src/cli.js

```diff
@@ -58,6 +58,19 @@ export function getConfig({useCLI=true} = {}) {
       type: 'boolean',
       default: false,
     })
+    .option('scan-file', {
+      alias: ['f'],
+      describe: 'Include only a single files in the scan',
```

Comment:
Should be "single file".

Actually I'd change this to:

`describe: 'Scan a single file',`